pallets/dia-oracle/src/lib.rs

```diff
@@ -87,7 +87,6 @@ pub mod pallet {
 	}
 
 	#[pallet::pallet]
-	#[pallet::generate_store(pub(super) trait Store)]
```

Comment:
What does this line do? Is this obsolete now?

Reply:
deprecated, https://github.com/paritytech/substrate/pull/13535